assets/js/modules/adsense/components/settings/SettingsForm.js

```diff
@@ -125,6 +126,8 @@ export default function SettingsForm() {
 			<AutoAdExclusionSwitches />
 
 			<AdBlockingRecoveryCTA />
+
+			<AdBlockingRecoveryToggle />
```

Comment:
⚠️ Ad Blocking recovery components shouldn't be rendered at all if the feature isn't enabled. We can address this in a follow-up issue since there are no critical side-effects invoked here, but something to keep in mind for all feature-conditional elements going forward. cc: @kuasha420